### ISSUE TITLE
USER STORY: Hyperparameter Optimization 

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/breast-cancer-detector#17]
As a **Developer** I can **adjust hyperparameters** so that **the model's prediction accuracy increases and meets the business requirements**.

**Acceptance Criteria**
* Adjust the hyperparameters using the validation set to get the accuracy to above 80%.
* Plot the changes in accuracy and loss for training and validation sets.
